src/back-end/lib/db/opportunity/code-with-us.ts

```diff
@@ -142,8 +144,6 @@ async function rawCWUOpportunityToCWUOpportunity(
     })
   );
 
-  delete raw.versionId;
```

Comment:
I'm trying to remember why we did it this way, and not deconstruct out like you are doing now. I think it may have been a linting issue (it didn't like the unused var), but if it works for you now, deconstructing is much nicer.